test/hmactest.c

```diff
@@ -275,19 +275,21 @@ static int test_hmac_copy_uninited(void)
     return res;
 }
 
-# ifndef OPENSSL_NO_MD5
+#ifndef OPENSSL_NO_MD5
```

Comment:
(comment) Yes, this is correct, but still wrong at lines 34, 87, and both before and after those - I probably wouldn't have fixed just this one - but I might have not changed this since it matches so many others. Still, someone else can fix all those later!

Reply:
Yep, there would be further layout mistakes on preprocessor directives to be fixed in this file,
and when seeing minor issues like this I'm inclined, too, to fix all of them along the way.
Yet I learned from from feedback on various bugfix PRs that these should include only the bare minimum of changes.